src/tools/create-file-dehydrator.js

```diff
@@ -0,0 +1,58 @@
+'use strict';
+
+const _ = require('lodash');
+
+const { DEFAULT_FILE_HYDRATOR_NAME } = require('../constants');
+const { DehydrateError } = require('../errors');
+const resolveMethodPath = require('./resolve-method-path');
+const wrapHydrate = require('./wrap-hydrate');
+
+const createFileDehydrator = input => {
+  const app = _.get(input, '_zapier.app');
+
+  const dehydrateFileFromRequest = (url, request, meta) => {
+    url = url || undefined;
+    request = request || undefined;
+    meta = meta || undefined;
+    if (!url && !request) {
+      throw new DehydrateError(
+        'You must provide either url or request arguments!'
+      );
+    }
+    return wrapHydrate({
+      type: 'file',
+      method: `hydrators.${DEFAULT_FILE_HYDRATOR_NAME}`,
+      bundle: { url, request, meta }
+    });
+  };
+
+  const dehydrateFileFromFunc = (func, inputData) => {
```

Comment:
The `fromRequest` and `fromFunc` threw me off a bit on first read cause it made me go "wait, we should be hydrating from a request or a func." I see now that it's communicating "dehydrate, with request as hydration options" or "dehydrate, with func as hydration options", but that doesn't make a very good method name. Unfortunately don't have anything better, so happy to roll with this for now.